acnportal/acnsim/models/tests/test_EVSE.py

```diff
@@ -76,11 +76,26 @@ def test_set_pilot_no_ev_invalid_rate(self):
 class TestFiniteRatesEVSE(TestEVSE):
     def setUp(self):
         self.evse = FiniteRatesEVSE('0001', [0, 8, 16, 24, 32])
+        self.evse2 = FiniteRatesEVSE('0002', [32, 16, 8, 24, 0])
```

Comment:
Define these EVSEs in each test function since they are only used by one test case each.

Reply:
Moved the EVSEs used by only one test case into the test functions using them, and redefined them as local variables rather than instance variables.